tests/conftest.py

```diff
@@ -414,3 +446,43 @@ def fixture_aggr_surfs_mean(fmurun_w_casemetadata, rmsglobalconfig, regsurf):
     os.chdir(origfolder)
 
     return (aggregated["mean"], metas)
+
+
+# ======================================================================================
+# Utilities
+# ======================================================================================
+
+
+def _parse_json(schema_path):
+    """Parse the schema, return JSON"""
+    with open(schema_path) as stream:
+        data = json.load(stream)
+
+    return data
+
+
+def _parse_yaml(yaml_path):
+    """Parse the filename as json, return data"""
+    with open(yaml_path, "r", encoding="utf-8") as stream:
+        data = yaml.safe_load(stream)
+
+    data = _isoformat_all_datetimes(data)
+
+    return data
+
+
+def _isoformat_all_datetimes(indate):
+    """Recursive function to isoformat all datetimes in a dictionary"""
+
+    if isinstance(indate, list):
+        indate = [_isoformat_all_datetimes(i) for i in indate]
+        return indate
+
+    if isinstance(indate, dict):
+        for key in indate:
+            indate[key] = _isoformat_all_datetimes(indate[key])
+
+    if isinstance(indate, (datetime.datetime, datetime.date)):
+        return indate.isoformat()
+
+    return indate
```

Comment:
It is still a bit "cumbersome" I see now...

First:
`resultdate = [_isoformat_all_datetimes(i) for i in indate]` (let `resultdate` be different from `indate`)

You could have made an `if, elif, else` instead and only `return resultdate` in the end.